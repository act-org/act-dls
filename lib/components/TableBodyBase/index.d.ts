/**
 * @prettier
 */
import * as React from 'react';
import { TableBodyProps } from '@material-ui/core';
export declare type Props = TableBodyProps;
declare const TableBodyBase: React.FC<Props>;
export default TableBodyBase;
//# sourceMappingURL=index.d.ts.map