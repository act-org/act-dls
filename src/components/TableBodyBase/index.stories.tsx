/**
 * @prettier
 */

import * as React from 'react';

import TableBodyBase from '.';

export const Base = (): React.ReactElement<any> => (
  <TableBodyBase>TableBody</TableBodyBase>
);

export default {
  component: TableBodyBase,
  parameters: {
    info: {
      text: 'This is the base variant of the TableBody component.',
    },
  },
  title: 'Atoms/TableBody',
};
