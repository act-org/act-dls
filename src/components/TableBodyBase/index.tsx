/**
 * @prettier
 */

import * as React from 'react';
import { TableBody, TableBodyProps } from '@material-ui/core';

export type Props = TableBodyProps;

const TableBodyBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <TableBody {...props} />;

export default TableBodyBase;
