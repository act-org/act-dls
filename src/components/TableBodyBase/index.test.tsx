/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import TableBodyBase from '.';

describe('TableBodyBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(<TableBodyBase>TableBody</TableBodyBase>);

    expect(container).toMatchSnapshot();
  });
});
