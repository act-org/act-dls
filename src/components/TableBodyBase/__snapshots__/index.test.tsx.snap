// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableBodyBase matches the snapshot 1`] = `
<div>
  <tbody
    class="MuiTableBody-root"
  >
    TableBody
  </tbody>
</div>
`;
